documents/manifesto.md

```diff
@@ -0,0 +1,28 @@
+# The Lua Users Foundation Manifesto
```

Comment:
I would add a note that this document is a draft and not yet normative

Reply:
I didn't add that because the fact that it is an unmerged PR is already indicating that it is a draft

Reply:
I suggest to add and merge. Unmerged PRs are less visible (and are harder to create a PR against).

Reply:
What?

Reply:
Please elaborate the question.

Reply:
A merged PR is not a PR anymore :thinking:

Reply:
I still do not understand.

Reply:
Okay, I understood what you mean

Etiene, please add "Draft" to the document, then it'll be merged. After that, others will make PRs with changes related to this document